NickvisionTubeConverter.Shared/Models/Download.cs

```diff
@@ -21,21 +21,13 @@ public enum Quality
     Resolution
 }
 
-/// <summary>
-/// Subtitle types for a download
-/// </summary>
-public enum Subtitle
-{
-    None = 0,
-    VTT,
-    SRT
-}
-
 /// <summary>
 /// A model of a media download
 /// </summary>
 public class Download
 {
+    private static string[] _youtubeLangCodes = { "af", "az", "id", "ms", "bs", "ca", "cs", "da", "de", "et", "en-IN", "en-GB", "en", "es", "es-419", "es-US", "eu", "fil", "fr", "fr-CA", "gl", "hr", "zu", "is", "it", "sw", "lv", "lt", "hu", "nl", "no", "uz", "pl", "pt-PT", "pt", "ro", "sq", "sk", "sl", "sr-Latn", "fi", "sv", "vi", "tr", "be", "bg", "ky", "kk", "mk", "mn", "ru", "sr", "uk", "el", "hy", "iw", "ur", "ar", "fa", "ne", "mr", "hi", "as", "bn", "pa", "gu", "or", "ta", "te", "kn", "ml", "si", "th", "lo", "my", "ka", "am", "km", "zh-CN", "zh-TW", "zh-HK", "ja", "ko" };
```

Comment:
I'd make this `internal static string[] YoutubeLangCodes` that why in `MediaUrlInfo` you can do `Download.YoutubeLangCodes` instead of making a second `_youtubeLangCodes`